indico/modules/events/management/templates/settings.html

```diff
@@ -4,7 +4,30 @@
 {% block title %}{% trans %}Settings{% endtrans %}{% endblock %}
 
 {% block content %}
+    {{ template_hook('event-actions', event=event) }}
+    {%- if event.room and event.room_reservation_links|length == 0 -%}
+        <div class="action-box warning">
+            <div class="section">
+                <div class="icon icon-bookmark"></div>
+                <div class="text">
+                    <div class="label">{% trans %}Book the event location{% endtrans %}</div>
+                    {% trans %}
+                        There is no booking linked to this event. Click the button on the right side to
```

Comment:
isn't "Click the button on the right side to book the location of this event." obvious when there's a "book the room" button next to it?